src/components/charts/SwipeChart.tsx

```diff
@@ -160,20 +179,8 @@ const SwipeChartComponent = (params: Props) => {
       minPriceLabelPosition.current = { x: minXPosition, y: minYPosition, origin: minXIndex > chartData.length / 2 ? 'right' : 'left' }
       maxPriceLabelPosition.current = { x: maxXPosition, y: maxYPosition, origin: maxXIndex > chartData.length / 2 ? 'right' : 'left' }
     }
-  }
-
-  // Min/Max Price Calcs
-  const prices = React.useMemo(() => chartData.map(dataPoint => dataPoint.y), [chartData])
-  const minPrice = Math.min(...prices)
-  const maxPrice = Math.max(...prices)
-
-  const sMinPriceString = useSharedValue(``)
-  const sMaxPriceString = useSharedValue(``)
-
-  const chartYRange = React.useMemo<[number, number]>(() => [minPrice - (maxPrice - minPrice) * 0.15, maxPrice], [minPrice, maxPrice])
-
-  const minPriceDataPoint = chartData.find(point => point.y === minPrice)
-  const maxPriceDataPoint = chartData.find(point => point.y === maxPrice)
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [minPriceDataPoint, maxPriceDataPoint, chartData, chartWidth, chartHeight])
 
   React.useEffect(() => {
     sMinPriceString.value = formatFiatString({ fiatAmount: minPrice.toString(), noGrouping: true, autoPrecision: true })
```

Comment:
Based on screenshot, this needs to add the currency symbol ($) and grouping separator.